pkg/minishift/cache/oc_caching_test.go

```diff
@@ -33,7 +34,7 @@ func TestIsCached(t *testing.T) {
 	setUp(t)
 	defer os.RemoveAll(testDir)
 
-	ocDir := filepath.Join(testDir, "cache", "oc", "v1.3.1")
+	ocDir := filepath.Join(testDir, "cache", "oc", tests.OPENSHIFT_VERSION)
```

Comment:
again, you make it work by changing the base version. The test would still fail with 1.3.1 now right?